### PR TITLE
Hello microverse setup

### PR DESCRIPTION
**Pull request by Fernando426**
- The broken link to the license was fixed
- Removed unnecessary comments from the readme file
- Removed unnecessary lines from the linter file, which are for checking JavaScript code

### CODE REVIEW

**Comment by Fernando426** [comment]
This is the link to the previous pull request:   https://github.com/Fernando426/Hello-world/pull/1

File: README.md [hunk 1]
```diff
@@ -1 +1,114 @@
...
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
```
Comment (abdulhamiid) [inline README.md]: - [ ] As requested by the previous reviewer, it is important that all main headers from the [README template](https://github.com/microverseinc/readme-template) are present in our repo to ensure that we have a well-descriptive readme file. 👍. The only optional section is the `FAQ`.

- [ ] Kindly check all the main headers of the README template below 👇 

image

- [ ] Your readme 👇 

image

Kindly add the section **Future Features**, including 1-3 potential features to be added. 👍 🛠

File: README.md [hunk 2]
```diff
@@ -1 +1,114 @@
...
+
+
+## 👥 Authors <a name="Fernando Montes"></a>
+
+
+👤 **Author1**
+
+- GitHub: [@Fernando426](https://github.com/Fernando426)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [Fernando Montes](https://www.linkedin.com/in/fernando-jose-montes-linares-523822270/?originalSubdomain=ve)
```
Comment (abdulhamiid) [inline README.md]: [OPTIONAL] - The link to your `Twitter` profile is broken, kindly fix it by adding the correct URL. 🛠

image

Let's avoid broken links in our readme file for a professional repository. 🤗👍

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @Fernando426 👋, 

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, I really appreciate the effort you put in! but there are still some which haven't been addressed yet.

almost-there

## Highlights 🌟

- ✔️ Descriptions are descriptive
- ✔️ Linters checks are passing
- ✔️ Use GitHub Flow

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Fernando426** [comment]
Hello, thanks for the review, about the instruction not to open a new pull
request, I have a question, should I just make the corrections and make a
new push?
 That way it will already be reflected in the current pull request?
Or do I need any additional steps?

**Review by ABDULALI3468 — APPROVED** [review]
Hi @Fernando426 , 👍 

# APPROVED 💯 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights 
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**